### PR TITLE
Now we use semver to verify the current version of the react-native package

### PR DESCRIPTION
**Pull request by digeff**
Now we use semver to verify the current version of the react-native package

### CODE REVIEW

File: src/common/reactNativeProjectHelper.ts [hunk 1]
```diff
@@ -27,19 +28,13 @@ export class ReactNativeProjectHelper {
...
-            if (components.length >= 2) { // Even though react-native versions have 3 components, we only care about the first 2
-                let minorVersion = parseInt(components[1], 10);
-                if (components[0] !== "0" || minorVersion >= 19) {
+        return new Package(this.workspaceRoot).dependencyPackage(ReactNativeProjectHelper.REACT_NATIVE_NPM_LIB_NAME).version()
+            .then(version => {
+                if (semver.gte(version, "0.19.0")) {
                     return Q.resolve<void>(void 0);
-                } else if (!(<any>Number).isNaN(minorVersion)) {
+                } else {
                     return Q.reject<void>(`Project version = ${version}`);
```
Comment (MSLaguana) [inline src/common/reactNativeProjectHelper.ts]: You should reject with an error not just a string

**Comment by MSLaguana** [comment]
You should also update the third party dependencies file so we don't forget later.

File: src/common/reactNativeProjectHelper.ts (hunk 1 above)
Comment (digeff) [inline src/common/reactNativeProjectHelper.ts]: I completely forgot that! Thanks! :)

**Comment by digeff** [comment]
I think that LCA generates that file. I can add a TODO in the third party dependencies file though. What do you think?

**Comment by dlebu** [comment]
LGTM!

**Pull request merged**